core/de.json

```diff
@@ -5,7 +5,7 @@
         "back": "Zurück zu Muonium",
         "favorites": "Favoriten",
         "login": "Login",
-        "notready": "Geschlossene Beta, willkommen zum Königreich der Wale",
+        "notready": "Private Beta, willkommen im Königreich der Wale",
```

Comment:
Yea okay... I just had some troubles with translations... Then perhaps there're some things to fix, I don't know.